Sources/SystemControl/SystemControl.swift

```diff
@@ -4,7 +4,7 @@ public protocol SystemControlValuesRetrieving {
     func getSystemValue<T: SystemControlValueDefining>(_ definition: T) throws -> T.ValueType
 }
 
-public struct SystemControl: SystemControlValuesRetrieving {
+public struct SystemControlProvider: SystemControlValuesRetrieving {
```

Comment:
Ah, I just realized that we already have `SystemControlValuesProviding` protocol and `SystemControlValuesProvider`. So maybe let's just leave `SystemControl` as is or rename it to e.g. `SystemControlDataSource`.

Reply:
We couldn't keep `SystemControl` because of the module namespace clash, that was the purpose of the change in the first place. Renamed to `SystemControlValuesRetriever`